lstchain/reco/dl0_to_dl1.py

```diff
@@ -140,7 +140,9 @@ def r0_to_dl1(input_filename = get_dataset_path('gamma_test_large.simtel.gz'),
               pedestal_path = None,
               calibration_path = None,
               time_calibration_path = None,
-              pointing_file_path = None
+              pointing_file_path = None,
+              ucts_t0 = None,
```

Comment:
Since these are floats, they should be `nan`, right?

Reply:
Right. Then should I defined them as `nan` here?
```
              ucts_t0 = nan,
              dragon_counter0 = nan
```

However, with those variables defined as `nan` I would need to change the condition `if ucts_t0 and dragon_counter0:` since it won't work as it is now.

```python
a = float('nan')
if a:
    print('yes')
else:
    print('no')
>>> yes

b = None
if b:
    print('yes')
else:
    print('no')
>>> no
```

Reply:
You can use `math.isnan(a)` or `np.isnan(a)` (first one is faster)

Reply:
I've defined those variables as `math.nan` in the call to `r0_to_dl1` function inside `dl0_to_dl1.py`. Should I define them by default as `math.nan` in the argparser of `lstchain_data_r0_to_dl1.py` script? Right now they are set to `None`.